src/App.js

```diff
@@ -20,6 +21,24 @@ function App() {
         >
           Learn React
         </a>
+
+        {/*download button, redownloads download icon on click*/}
+        <Button>
+        <a href='download_icon.png' download>
```

Comment:
to better componentize, pass the link and icon as props https://reactjs.org/docs/components-and-props.html

Reply:
the link needs to wrap the image for the image to actually be a link!